### PR TITLE
[1.1.0] Add sample commands for ELK installation

### PR DESCRIPTION
**Pull request by sgayangi**

### CODE REVIEW

**Review by CrowleyRajapakse — APPROVED** [review]

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md [hunk 1]
```diff
@@ -70,8 +70,47 @@ You can also set multiple publishers for analytics as follows. Replace ```choreo
...
+### Installing Elasticsearch
+
+1. Apply the following commands to install Elasticsearch.
+
+```bash
+helm repo add elastic https://helm.elastic.co
+helm repo update
+helm install elasticsearch elastic/elasticsearch
+```
+2. Wait for the pods to spin up. You can view them using the following command.
```
Comment (Tharanidk) [inline en/docs/setup/analytics/configure-analytics-for-elk-stack.md:84]: add a line break after this line

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md [hunk 2]
```diff
@@ -70,8 +70,47 @@ You can also set multiple publishers for analytics as follows. Replace ```choreo
...
+```bash
+kubectl get pods
+```
+3. Once the pods all have spun up, you can get the elasticsearch credentials using the following command.
```
Comment (Tharanidk) [inline en/docs/setup/analytics/configure-analytics-for-elk-stack.md:88]: Add a blank line before and after a bullet point to increase readability. Same for bullet no.4 also.

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md [hunk 3]
```diff
@@ -70,8 +70,47 @@ You can also set multiple publishers for analytics as follows. Replace ```choreo
...
+```
+
+### Installing Kibana
+
+1. Apply the following commands to install Kibana.
+
+```bash
+helm install kibana elastic/kibana 
+```
+2. Wait for the pods to spin up. You can view them using the following command.
```
Comment (Tharanidk) [inline en/docs/setup/analytics/configure-analytics-for-elk-stack.md:104]: Add a blank line before and after a bullet points

Comment (sgayangi) [reply en/docs/setup/analytics/configure-analytics-for-elk-stack.md:104]: Fixed in PR #583

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md (hunk 2 above)
Comment (sgayangi) [reply en/docs/setup/analytics/configure-analytics-for-elk-stack.md:88]: Fixed in PR #583

File: en/docs/setup/analytics/configure-analytics-for-elk-stack.md (hunk 1 above)
Comment (sgayangi) [reply en/docs/setup/analytics/configure-analytics-for-elk-stack.md:84]: Fixed in PR #583

**Pull request merged**